bundles/io/org.openhab.io.jetty.certificate/src/main/java/org/openhab/io/jetty/certificate/internal/CertificateGenerator.java

```diff
@@ -0,0 +1,198 @@
+package org.openhab.io.jetty.certificate.internal;
```

Comment:
No, the one from all other source files: https://github.com/openhab/openhab2/blob/master/bundles/core/org.openhab.core/src/main/java/org/openhab/core/OpenHAB.java#L1